{
	"id": "CVE-2021-36979",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-36979",
				"description": "Unicorn Engine 1.0.2 has an out-of-bounds write in tb_flush_armeb (called from cpu_arm_exec_armeb and tcg_cpu_exec_armeb).",
				"references": [
					{
						"source": "CVE",
						"url": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=30391"
					},
					{
						"source": "CVE",
						"url": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/unicorn/OSV-2020-2305.yaml"
					},
					{
						"source": "CVE",
						"url": "https://github.com/unicorn-engine/unicorn/commit/bf1713d9e011b55ca1f502a6779fc4722b4bb077"
					},
					{
						"source": "CVE",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MZ6LOCJXHQVU6SCJLFDJINBOVJYYENLX/"
					},
					{
						"source": "mitre",
						"url": "https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=30391"
					},
					{
						"source": "mitre",
						"url": "https://github.com/google/oss-fuzz-vulns/blob/main/vulns/unicorn/OSV-2020-2305.yaml"
					},
					{
						"source": "mitre",
						"url": "https://github.com/unicorn-engine/unicorn/commit/bf1713d9e011b55ca1f502a6779fc4722b4bb077"
					},
					{
						"source": "mitre",
						"url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/MZ6LOCJXHQVU6SCJLFDJINBOVJYYENLX/"
					}
				],
				"modified": "2024-08-04T01:09:07.632Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2021/CVE-2021-36979.json"
		]
	}
}
